lib/OnyxConnectionManager.ts

```diff
@@ -229,6 +238,15 @@ class OnyxConnectionManager {
         this.connectionsMap.clear();
     }
 
+    /**
+     * Refreshes the connection manager's session ID. Used in Onyx.clear() in order to
+     * create new connections after Onyx is cleared, instead of reusing existing connections
+     * that can produce unexpected behavior in Onyx subscribers.
```

Comment:
Can we get more specific? What is the "unexpected behavior"? I think including an example will help later on if someone needs to make changes here or questions whether it is still required.

Reply:
I moved all explanations to `private sessionID: string;`, please have a look!